lib/VertexArrayObject.js

```diff
@@ -212,3 +212,24 @@ VertexArrayObject.prototype.draw = function(type, size, start)
 
 	return this;
 }
+
+/**
+ * Destroy this vao
+ */
+VertexArrayObject.prototype.destroy = function()
+{
+	// lose references
+	this.gl = null;
+	this.indexBuffer = null;
+	this.attributes = null;
+	this.nativeState = null;
+	this.unbind();
```

Comment:
Looks good man! One tiny tweak for ya - We don't need the unbind, other than that this is all good!